packages/react-components/react-toast/src/state/useToaster.ts

```diff
@@ -42,13 +45,21 @@ export function useToaster() {
   );
 
   return {
-    isToastVisible: (toastId: ToastId) => {
-      if (toaster) {
-        return toaster.isToastVisible(toastId);
-      }
-
-      return false;
-    },
+    toasterRef,
+    isToastVisible: toaster.isToastVisible,
     getToastsToRender,
   };
 }
+
+function useDefaultToastOptions(options: DefaultToastOptions) {
+  const { pauseOnHover, pauseOnWindowBlur, position, timeout } = options;
+  return React.useMemo<DefaultToastOptions>(
+    () => ({
+      ...(pauseOnHover && { pauseOnHover }),
+      ...(pauseOnWindowBlur && { pauseOnWindowBlur }),
+      ...(position && { position }),
+      ...(timeout && { timeout }),
+    }),
```

Comment:
```suggestion
    () => ({
      pauseOnHover,
      pauseOnWindowBlur,
      position,
      timeout,
    }),
```

And nothing happened 🤨